### PR TITLE
Cooking and Drinks Supplemental Update

### LINKED ISSUE #1733: Tequila Sunrise is invisible.
## Description
Tequila Sunrise sprite is gone.

Also as a sidenote you can't feed xenos drinks, very sad.
## Difference between expected and actual behavior
No Tequila sunrise sprite, there should be.

## Steps to reproduce
Step 1. Attempt to make Tequila Sunrise
Step 2. Watch in awe as it disappears from your hand (no sprite)

## Issue bingo (not required to be all checked)

- [x] Issue could be reproduced at least once
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] Couldn't find an existing issue about this (https://github.com/cmss13-devs/cmss13/issues)

### PR DESCRIPTION
**Pull request by Unknownity**
## About The Pull Request

This PR adds in additional features for the Mess Technician to mess with and a general "quality of life" upgrades of the current state of cooking and making drinks.
Soda and Booze dispensers have been fixed to be able to wrenchable now (a previous PR in gitlab era tried making them wrenchable but forgot a crucial detail), with one exception currently being the soda dispenser in the Almayer main kitchen (not officer's mess) since its bolted to the wall.

The soda dispenser has been upgraded to include additional drinks, these being tomato,carrot,berry,grape,lemon and banana juices. Ice Tea has been removed from the dispenser since it is an easily made mixed drink (tea+ice). The secret mode of the soda dispenser has been adjusted to keep up with the theme of being a non-alcoholic dispenser (previously it gave you an alcoholic drink and an entire can of grape soda), If you want to spoil yourself on what it dispenses now, check the code.

The booze dispenser has been adjusted with what reagents it dispenses, all the non alcoholic liquids except for very basic ones (water,sodawater,sugar,ice) have been removed as you can find them in the soda dispenser and it gained two new drinks called thirteenloko which has been previously the secret drink of the soda dispenser and tequila. The booze dispenser has obtained an additional secret drink.

The groceries variant of the fridge has obtained new ingredients inside, these being: cherries, limes, lemons, oranges and bananas. So the Mess Technician isn't forced to grow these at the start when they could very easily exist in the fridge.

The Officer's Mess meat fridge has gained some carp fillets instead of already made tofu products, Officers should obviously have some exotic meat right?

A new condiment has surfaced, it is a cold sauce bottle that can be found in Shivas Snowball (it's made from there) and in the USS Almayer's main kitchen freezer crate. the sprite is simply a resprite of one of the hot sauce bottles.

some drinks lacked sprites (were broken) such as Ale and Tequila Sunrise, probably weren't the only ones but so far only these two were found and fixed. Ale's got its bottled sprite back thanks to Bay12
fixes #1733 

A new alcoholic drink was added, the Mojito. we already had an unused sprite in the repo for it, so I just added it fully in the game.

someone put the wrong id for universal enzyme in the primordial cheese, so fixed that.

## Why It's Good For The Game

Provides additional options for the Mess Technician to play with, removes the necessity to grow very common fruit right at the start just because there's lack of any in the fridge. Makes Mess Technicians also able to bring their soda/booze dispensers from the Colony Bars or Almayer's Officer Mess down to the FOB

## Changelog

:cl: Unknownity
add: The Soda and Booze dispensers have obtained additional drinks.
add: The grocery fridge has been filled with more unique fruit and vegetables.
add: The Officer's Mess meat fridge has been filled with carp meat.
add: A new condiment bottle has surfaced from Shivas Snowball, you can find it there and on the USS Almayer inside the main kitchen's freezer crate.
add: Added a new alcoholic drink, the Mojito! The recipe to make it is roughly the same as in real life.
fix: Soda and Booze dispensers are now actually wrenchable.
fix: Fixed certain drinks not having any sprites.
fix: Fixed Primordial Cheese not containing universal enzyme.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry_machinery/chem_dispenser.dm [hunk 1]
```diff
@@ -239,12 +254,26 @@
...
+		if(do_after(user, 20, INTERRUPT_ALL|BEHAVIOR_IMMOBILE, BUSY_ICON_BUILD))
+			if(!src) return
+			playsound(src.loc, 'sound/items/Ratchet.ogg', 25, 1)
+			switch (anchored)
+				if (0)
+					anchored = 1
+					user.visible_message("[user] tightens the bolts securing \the [src] to the surface.", "You tighten the bolts securing \the [src] to the surface.")
+				if (1)
+					user.visible_message("[user] unfastens the bolts securing \the [src] to the surface.", "You unfasten the bolts securing \the [src] to the surface.")
+					anchored = 0
```
Comment (stanalbatross) [inline code/modules/reagents/chemistry_machinery/chem_dispenser.dm:278]: TRUE FALSE please, also can't you just call parent?

File: code/modules/reagents/chemistry_machinery/chem_dispenser.dm [hunk 2]
```diff
@@ -202,23 +202,38 @@
...
```
Comment (stanalbatross) [inline code/modules/reagents/chemistry_machinery/chem_dispenser.dm:235]: same

**Review by stanalbatross — CHANGES_REQUESTED** [review]
the rest is fine

**Review by stanalbatross — APPROVED** [review]

**Review by kugamo — APPROVED** [review]
Sprites are fine.

**Comment by moltoretardo** [comment]
Add that it fixes #1733

**Comment by Unknownity** [comment]
> Add that it fixes #1733
In the about: "some drinks lacked sprites (were broken) such as Ale and Tequila Sunrise, probably weren't the only ones but so far only these two were found and fixed. Ale's got its bottled sprite back thanks to Bay12"

and changelog "fix: Fixed certain drinks not having any sprites." as it contains multiple.

**Pull request merged**